main.css

```diff
@@ -120,6 +135,12 @@ a {
 .justify-content-between {
   justify-content: space-between;
 }
+.justify-center {
```

Comment:
There are `justify-items` and `justify-content` properties, so let's rename the class to `justify-content-center` to make it more specific.